### PR TITLE
HOCS-4259: increase logging around createStage team/user assignment

### PR DESCRIPTION
**Pull request by gary-newbolt**

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java [hunk 1]
```diff
@@ -509,25 +509,26 @@ public void createCaseConversionNote(String caseUUIDString, String stageUUIDStri
 
     private UUID deriveTeamUUID(String caseUUIDString, String stageTypeString, String allocationTeamString) {
         UUID teamUUID;
-        if (StringUtils.isEmpty(allocationTeamString)) {
+        if (!StringUtils.hasText(allocationTeamString)) {
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java:512]: Is it worth logging out this allocationTeamString as well?

File: src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java [hunk 2]
```diff
@@ -509,25 +509,26 @@ public void createCaseConversionNote(String caseUUIDString, String stageUUIDStri
...
         return teamUUID;
 
     }
 
     private UUID deriveUserUUID(String caseUUIDString, String stageTypeString, String allocatedUserId) {
-        UUID userUUID = null;
-        if (!StringUtils.isEmpty(allocatedUserId)) {
-            log.debug("Assigning user {} to stage {} for case {}", allocatedUserId, stageTypeString, caseUUIDString);
-            userUUID = UUID.fromString(allocatedUserId);
+        if (StringUtils.hasText(allocatedUserId)) {
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java:525]: Same here, worth logging out the allocatedUserId?

File: src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java (hunk 1 above)
Comment (gary-newbolt) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java:512]: In this one... that if block is only called if the `allocatedTeamString` is `null` anyway and in both the subsequent log output it logs the resolved teamUUID, specifically the new final info level one. From the logic there's no way if can escape that method as null, if the `infoClient` call fails it should trigger the global handling anyway.

File: src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java (hunk 2 above)
Comment (gary-newbolt) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java:525]: I think the same is true here... if there is an `allocatedUserId` it's logged out in the next line. If there isn't, there's a `warn` level message explicitly stating it.

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**